modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm

```diff
@@ -0,0 +1,237 @@
+/datum/armament_entry/company_import/deforest
+	category = DEFOREST_MEDICAL_NAME
+	company_bitflag = CARGO_COMPANY_DEFOREST
+
+// Basic first aid supplies like gauze, sutures, mesh, so on
+
+/datum/armament_entry/company_import/deforest/first_aid
```

Comment:
it'd be nice if these were a little more "in bulk" rather than 6 orders of 1 (one) suture

Reply:
You can order as many as you want, and they'll all show up in the same container regardless. If you want to get these in bulk (I'm not sure of toooo many situations where that's wanted) then cargo can just cope with the long purchased order log